### ISSUE TITLE
No border of the image when you set up transparent background

### ISSUE THREAD

**Issue by pervoj** [maoschanz/drawing#299]
## Description
When you set up transparent background, you don't see border of the image. It is very impractical. Add for example some black border around the image.

Here is one screenshot. You can see the border of the image on the end of the red lines.
![Screen shot](https://user-images.githubusercontent.com/71781857/98852715-5c592480-2458-11eb-8dca-30830a3571ad.png)

### Steps to reproduce the bug
The bug appears when you set up transparent background.

## System
- Device (desktop computer/tablet/phone/...): Fujitsu Lifebook S761
- OS: Linux Mint 20 Ulyana
- Desktop environment: Cinnamon
- Package format (flatpak, native package, snap?): native package (apt)
- App version: 0.4.13

PS: Sorry for my bad English... :)

**Comment by pervoj** [comment]
You can use also this pattern:
![Pattern](https://user-images.githubusercontent.com/71781857/98952257-8879b180-24fb-11eb-9a53-ede4127e3937.png)
I think it's better than border.

**Comment by maoschanz** [comment]
i'll just do a border, because this way it also helps with #184  ![Capture d’écran de 2021-06-24 16-14-42](https://user-images.githubusercontent.com/19996113/123278545-51469680-d507-11eb-9112-0ecfbae71640.png)
